gdbgui/src/js/Threads.jsx

```diff
@@ -115,7 +115,7 @@ class Threads extends React.Component {
     }
     return (
       <span key={'thread' + thread.id} className={`${cls}`} style={{fontSize: '0.9em'}}>
-        {status} {Memory.make_addrs_into_links_react(thread['target-id'])}, core {thread.core}, {thread.state}, id {thread.id}
+        {status} {Memory.make_addrs_into_links_react(thread['target-id'])}, core {thread.core}, {thread.state}, id {thread.id}, name {thread.name}
```

Comment:
I am looking at some sample output and don't see a "name" field on the thread. Which version of gdb are you running?

Reply:
GNU gdb (GDB) 8.1
maybe it should be an optional then